README.md

```diff
@@ -1,6 +1,11 @@
 [![Build Status](https://travis-ci.com/Coding-Coach/find-a-mentor.svg?branch=master)](https://travis-ci.com/Coding-Coach/find-a-mentor)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/Coding-Coach/find-a-mentor/issues)
 
+<span class="badge-patreon"><a href="https://www.patreon.com/codingcoach_io" title="Donate to this project using Patreon"><img src="https://img.shields.io/badge/patreon-donate-yellow.svg" alt="Patreon donate button" /></a></span>
+
+><a href="https://patreon.com/codingcoach_io"><img src="https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.herokuapp.com%2Fendel&style=for-the-badge" /> </a>
```

Comment:
I'm not sure I'm following. Do we have 2 links now?

Reply:
I am asking which we prefer, so I can remove on type and retain the other one

Reply:
I'll vote for B :)

Reply:
I have pushed updates